.vscode/settings.json

```diff
@@ -21,7 +21,7 @@
     "files.insertFinalNewline": false,
     "files.trimTrailingWhitespace": false
   },
-  "rust-analyzer.linkedProjects": ["nextpack/Cargo.toml"],
+  // "rust-analyzer.linkedProjects": ["nextpack/Cargo.toml"],
```

Comment:
Fixship: I don't think this PR hsould make this change, but the turbopack team is removing this @jridgewell